docs/source/image_process.mdx

```diff
@@ -0,0 +1,127 @@
+# Process image data
+
+🤗 Datasets support loading and processing images with the [`~Image`] feature. This guide will show you how to:
+
+- Load an image dataset.
+- Load a generic image dataset with `ImageFolder`.
+- Add data augmentations to your images.
+- Use [`~Dataset.map`] to quickly apply your transforms to the entire dataset.
+
+## Image datasets
+
+The images in an image dataset are typically either a:
+
+- PIL `image`.
+- Path to an image file that you can load. 
+
+For example, load the [Food-101](https://huggingface.co/datasets/food101) dataset and take a look:
+
+```py
+>>> from datasets import load_dataset, Image
+
+>>> dataset = load_dataset("food101", split="train[100:]")
+>>> dataset[0]
+{'image': <PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=384x512 at 0x7FC45AB5C590>,
+ 'label': 6}
+```
+
+The [`~Image`] feature automatically decodes the data from the `image` column to return an image object. Now try and call the `image` column to see what the image is:
+
+```py
+>>> from datasets import load_dataset, Image
+
+>>> dataset = load_dataset("food101", split="train[100:200]")
+>>> dataset[0]["image"]
+```
+
+![image_process_beignet](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/image_process_beignet.png)
+
+
+To load an image from its path, use the [`~Dataset.cast_column`] method. The [`~Image`] feature will decode the data at the path to return an image object:
+
+```py
+>>> from datasets import load_dataset, Image
+
+>>> dataset = load_dataset("cats_vs_dogs").cast_column("image_file_path", Image())
+>>> dataset[0]["image_file_path"]
+```
+
+![image_process_dog](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/image_process_dog.png)
+
+<Tip>
+
+You can also access the path and bytes of an image file by setting `decode=False` when you load a dataset. In this case, you will need to cast the `image` column:
+
+`dataset = load_dataset("food101", split="train[100:]").cast_column('image', Image(decode=False))`
+
+</Tip>
+
+## ImageFolder
```

Comment:
WDYT about deleting [this ImageFolder section](https://huggingface.co/docs/datasets/master/en/loading#image-folders) to have only this one, because ImageFolder is not as generic loader as CSV, JSON. It's a bit more specific and associated with CV. 

cc @lhoestq for his opinion on this

Reply:
Yes I agree, let's not duplicate the documentation. Feel free to redirect users that are looking for vision-specific formats from `loading.mdx` to `image_process.mdx` though ^^